WordPress/src/main/java/org/wordpress/android/ui/menus/MenusSpinner.java

```diff
@@ -128,7 +128,7 @@ public View getView(int position, View convertView, ViewGroup parent) {
             }
 
             TextView titleView = (TextView) convertView.findViewById(R.id.menu_spinner_title);
-            titleView.setText(String.format(mTitle, getCount()));
```

Comment:
oh - there's a problem with this, as the component is used for Menu Locations as well - maybe we can pass a `int baseCount` parameter in the creator method for MenuSpinner, or maybe in `setItems` that can be used as a subtractor?